### PR TITLE
Entertainer Role [Ready]

### PR DESCRIPTION
**Pull request by Max-023**
## About The Pull Request

Adds a "new" job, the entertainer! This snazzy new job is for those who wish to chill with the crew and....well.....entertain them! With plenty of alt titles such as dancer and magician, there is plenty of fun to be had and laughs to be made! It even works well for an entry level role for those that want to be a mime or clown, but don't have the street cred!

## Why It's Good For The Game

Removes weight from the visitor role, as entertainer and morale officer were alt titles of it, and instead gives it to a new role. Tether already has an entertainment area, but no jobs to go with it. Additionally, it does offer a candidacy role for those that do want to get the whitelist for the clown and mime role, as it provides a similar experience for them, but with added flexibility.

## Changelog
:cl:
add: Entertainer role
delete: Removes the entertainer and morale officer alt titles from visitor
/:cl:

### CODE REVIEW

File: code/game/jobs/jobs.dm [hunk 1]
```diff
@@ -35,6 +35,7 @@ var/const/BOTANIST			=(1<<2)
 var/const/CHEF				=(1<<3)
 var/const/JANITOR			=(1<<4)
 var/const/LIBRARIAN			=(1<<5)
+var/const/ENTERTAINER		=(1<<5)
```
Comment (TheLordME) [inline code/game/jobs/jobs.dm:38]: Change it to 
`=(1<<15)`
the 5 is already used by librarian, and that results in the issue with the job pref

**Comment by TheLordME** [comment]
**To give a more indepth explaination of my discord message:**
Unless you want the entertainer to have a specific unique collection of access, just add them to a job that already has the access you want as an alt_title.
If you want to give them specific equipment you can orient yourself at the detective/forensic technician code.
If the job you would add them to has to little slots, add more slots. 
There is no point in adding a second no access job.
There is currently no entertaining area on the triumph.

**For your issue:**
See my comment on: `code/game/jobs/jobs.dm - Line 38`

**Future issue youll probably have:**
You need to define the entertainer outfit somewhere. Otherwise youll have some issue with the spawned character

**Comment by Max-023** [comment]
There is an entertainment section on the Tether, currently, so this would pair well with when the tether rolls out. As for pairing it with access relations, I do plan on adding an entertainer storage area for random items they could use, but as of right now, as far as I know, editing the ship is off limits.

**Comment by silicons** [comment]
#3013

**Comment by Captain277** [comment]
Gonna go ahead and close this since Visitor has been restored.

**Pull request closed without merging**